test/integration/users.test.js

```diff
@@ -357,6 +358,20 @@ describe("Users", function () {
       expect(previousPageResponse.body.links).to.have.property("prev");
       expect(previousPageResponse.body.users).to.have.length(2);
     });
+
+    it("should return all the user who are member and not archieved if members=true is sent in the query", async function () {
+      const response = await chai.request(app).get(`/users?member=true`).set("cookie", `${cookieName}=${jwt}`);
+
+      expect(response).to.have.status(200);
+      expect(response.body).to.be.a("object");
+      expect(response.body.message).to.equal("Users returned successfully!");
+      expect(response.body).to.have.property("links");
+      expect(response.body.links).to.have.property("next");
+      expect(response.body.links).to.have.property("prev");
+
+      expect(response.body.users[0].roles.member).to.be.equal(true);
+      expect(response.body.users[0].roles.archived).to.be.equal(false);
```

Comment:
can we also test the length of the response returned?

Reply:
I think we do not need to test it